src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -899,7 +899,7 @@ const formConfig = {
                   'intendsToEnroll',
                   'over18HelplessChild',
                 ]),
-                otherStatus: { type: 'string' },
+                _unused: { type: 'string' },
```

Comment:
this value isn't actually read or collected, but it's included because of the parent custom page being used here. Changing this key to `_unused` so there's less confusion.